CMakeLists.txt

```diff
@@ -195,6 +195,14 @@ endif ()
 
 option(ADD_WINDOWS_ENCLAVE_TESTS "Build Windows enclave tests" OFF)
 
+# NOTE: Building OpenEnclave using snmalloc for memory allocation is an experimental option.
+# Longer term, there will be a pluggable allocator mechanism allow users to chose from a
```

Comment:
grammar: "allow" -> "to allow"
"chose" -> "choose"

Reply:
Fixed